specification.md

```diff
@@ -1,15 +1,279 @@
 # Specification
 
+This document serves as the 0.1 specification for SBOMit (SBOM on in-toto).
+The specification should not be considered stable at this time.
+
 ## Introduction
 
+### Motivation
+A major problem with the state of an SBOM today is that the documents are 
+often inaccurate.  This is largely because the documents are derived by looking
+at the resulting software and trying to understand what happened in the past.
+
+This specification proposes a means to generate metadata for an SBOM while the
+software is being created.  Furthermore, the means by which this information
+is captured uses (in-toto)[https://in-toto.io] attestations and layouts.  This 
+provides cryptograpic validation that this information is correct, handles 
+key distribution and management to indicate which parties should be trusted
+for each step, and captures information about the environment in which the 
+steps are run.  
+
+As a result, using SBOMit provides a more accurate SBOM when parties are 
+honest.  When malicious parties interfere in the process, SBOMit provides 
+a mix of traceability (knowing which party was malicious) and prevention 
+(blocking malicious software from being trusted), depending on how the 
+in-toto steps are configured..
+
+
+### Definitions
+
+SIT -- An SBOM which is derived from a SBOMit document.  It can
```

Comment:
Interesting: by "derived," do you mean that an SBOMIT document is something like a high-level intermediate representation from SBOMs that can then be used to generate low-level "machine code," so to speak?

Reply:
Yes, you can think of it this way.  It is like the complete "source" document which contains things like in-toto metadata which will not be in the SBOM (SIT).